docs/guide/api/about-api-documentation.rst

```diff
@@ -22,6 +22,13 @@ While these tools help you with the technical details, they don't create good de
 
 .. _documentation-driven-design:
 
+Test-Driven Documentation
+-------------------------
+
+Test-driven documentation aims to improve upon the typical approaches to automated documentation. It allows you to write the bulk of the documentation by hand while also ensuring its accuracy by using your API's tests to generate some of the content.
+
+Projects such as `Spring REST Docs <https://spring.io/projects/spring-restdocs>`_ use your API's tests to generate small snippets of documentation that can be included in hand-written API documentation. The accuracy of the snippets is guaranteed by using your API's tests to generate them – if the snippets become inaccurate, the API's tests will fail.
+
```

Comment:
Thanks for the review. I don't think you've missed something as the wording was ambiguous. I've force-pushed a new commit that hopefully makes things clearer